### ISSUE TITLE
` forge install latest` show some warning 

### ISSUE THREAD

**Issue by mave99a** [ArcBlock/forge-cli#78]
` forge install latest` show some warning if run immediately after npm install.

```
Examples:

  Please install a forge-release before running any other commands
  > forge install latest
  > forge install --mirror http://arcblock.oss-cn-beijing.aliyuncs.com

  Curious about how to use a subcommand?
  > forge help install

```

From the install finish message it's very easy to try to run `forge install latest`

```
maos-MacBook-Pro:Downloads mao$ forge install latest
✔ initialized tmp dir for forge-cli: /Users/mao/.forge_cli/tmp
✔ initialized bin dir for forge-cli: /Users/mao/.forge_cli/bin
✔ initialized cache dir for forge-cli: /Users/mao/.forge_cli/cache
✔ initialized release dir for forge-cli: /Users/mao/.forge_cli/release
(node:51592) UnhandledPromiseRejectionWarning: TypeError: Invalid Version: 
    at new SemVer (/usr/local/lib/node_modules/@arcblock/forge-cli/node_modules/semver/semver.js:323:11)
    at compare (/usr/local/lib/node_modules/@arcblock/forge-cli/node_modules/semver/semver.js:614:39)
    at Function.gt (/usr/local/lib/node_modules/@arcblock/forge-cli/node_modules/semver/semver.js:643:10)
    at checkUpdate (/usr/local/lib/node_modules/@arcblock/forge-cli/src/core/env.js:676:14)
    at setupEnv (/usr/local/lib/node_modules/@arcblock/forge-cli/src/core/env.js:76:3)
    at async Command.<anonymous> (/usr/local/lib/node_modules/@arcblock/forge-cli/src/core/cli.js:69:7)
(node:51592) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:51592) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
maos-MacBook-Pro:Downloads mao$ forge version
✖ required config file /Users/mao/.forge_cli/release/forge/release.yml not found
ℹ if you have not setup forge yet, please run forge install first
maos-MacBook-Pro:Downloads mao$ forge install

```

`forge install` works.

**Comment by sunboshan** [comment]
Same issue, the first time it's running `forge install` will produce above error. Then do it again, it will work.

```
➜ forge install
✔ initialized tmp dir for forge-cli: /Users/boshansun/.forge_cli/tmp
✔ initialized bin dir for forge-cli: /Users/boshansun/.forge_cli/bin
✔ initialized cache dir for forge-cli: /Users/boshansun/.forge_cli/cache
✔ initialized release dir for forge-cli: /Users/boshansun/.forge_cli/release
(node:59662) UnhandledPromiseRejectionWarning: TypeError: Invalid Version:
    at new SemVer (/usr/local/lib/node_modules/@arcblock/forge-cli/node_modules/semver/semver.js:323:11)
    at compare (/usr/local/lib/node_modules/@arcblock/forge-cli/node_modules/semver/semver.js:614:39)
    at Function.gt (/usr/local/lib/node_modules/@arcblock/forge-cli/node_modules/semver/semver.js:643:10)
    at checkUpdate (/usr/local/lib/node_modules/@arcblock/forge-cli/src/core/env.js:676:14)
    at setupEnv (/usr/local/lib/node_modules/@arcblock/forge-cli/src/core/env.js:76:3)
    at process._tickCallback (internal/process/next_tick.js:68:7)
    at Function.Module.runMain (internal/modules/cjs/loader.js:744:11)
    at startup (internal/bootstrap/node.js:285:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:739:3)
(node:59662) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:59662) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

~ took 4s
➜ forge install

██████╗ ██╗   ██╗     █████╗ ██████╗  ██████╗██████╗ ██╗      ██████╗  ██████╗██╗  ██╗
██╔══██╗╚██╗ ██╔╝    ██╔══██╗██╔══██╗██╔════╝██╔══██╗██║     ██╔═══██╗██╔════╝██║ ██╔╝
██████╔╝ ╚████╔╝     ███████║██████╔╝██║     ██████╔╝██║     ██║   ██║██║     █████╔╝
██╔══██╗  ╚██╔╝      ██╔══██║██╔══██╗██║     ██╔══██╗██║     ██║   ██║██║     ██╔═██╗
██████╔╝   ██║       ██║  ██║██║  ██║╚██████╗██████╔╝███████╗╚██████╔╝╚██████╗██║  ██╗
╚═════╝    ╚═╝       ╚═╝  ╚═╝╚═╝  ╚═╝ ╚═════╝╚═════╝ ╚══════╝ ╚═════╝  ╚═════╝╚═╝  ╚═╝

ℹ Detected platform is: darwin
✔ Latest forge release version: v0.32.2
✔ Release asset info fetch success forge_darwin_amd64.tgz
rm: no such file or directory: /Users/boshansun/.forge_cli/tmp/forge_darwin_amd64.tgz
```

**Comment by wangshijun** [comment]
this is fixed in this commit: https://github.com/ArcBlock/forge-cli/pull/75/commits/9f15a9279bd1e96bae6d0a701902399c936da380, because I accidently removed cli version setting in release v0.32.4.

**Comment by wangshijun** [comment]
the fix should land in forge-cli v0.32.5
